### ISSUE TITLE
[Bug]: missing window manager buttons

### ISSUE THREAD

**Issue by Jopp-gh** [FreeTubeApp/FreeTube#8706]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

1. launch freetube
2. there are no window manager buttons on-screen (minimize, expand, close)
3. the only way to close freetube is to use a shortcut, eg. `ctrl+w`
4. after closing freetube I am unable to re-launch freetube, please see bug report #8705 

### Expected Behavior

freetube should have window manager buttons like any other app. I don't want any app to be fullscreen and on top all-times. If I watch movies, then a simple press on F is enough. Freetube's window doesn't have to be fullscreen. 

### Issue Labels

accessibility issue

### FreeTube Version

v0.23.13 Beta

### Operating System Version

Linux Mint Cinnamon 21.3

### Installation Method

Flathub

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.22.x

### Additional Information

since window manager buttons aren't visible, freetube is "always on top" and makes any application switch difficult

- Again I need to use shortcuts and can't use my mouse to switch from one app to another. 
- Also I can't access my system tray because freetube is basically in fullscreen and on top over everything else on screen. pressing S makes videos inside freetube fullscreen but it doesn't scale freetube's window.
- freetube is the only flatpak without window manager buttons, but this hasn't been always like this, eg. v22.x had wm buttons. 

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
FreeTube remembers the windows state, so if you were in fullscreen last time you used FreeTube it will reopen in fullscreen the next time you start the app too. Press F11 to exit fullscreen (this is already covered in existing issues please search through both **open and closed** issues next time as instructed by the issue template).
